e2e/scenario_test.go

```diff
@@ -632,6 +633,23 @@ func Scenario_ubuntu2204(t *testing.T) {
 	})
 }
 
+func Scenario_ubuntu2204WireServerBlock(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "Tests that a node using the Ubuntu 2204 VHD can be properly bootstrapped",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				FileHasContentsValidator("/opt/azure/containers/kubelet.sh", "iptables -I FORWARD -d 168.63.129.16 -p tcp -m multiport --dports 80,32526 -j DROP"),
```

Comment:
should we just add this as a common validator instead of having an extra scenario for it?

Reply:
We could and we probably should, but I think if we wanted to do this it would make more sense to figure out "must have" things like this one and group them into a common validator that runs on all tests

Reply:
Happy to work on it later on too.

Reply:
spoke offline, sounds like we can just add this to list of the common validators and include a real network check from inside the pod network to verify the ports are blocked